osquery/main/windows/main.cpp

```diff
@@ -79,7 +79,7 @@ static void UpdateServiceStatus(DWORD controls,
 static auto kShutdownCallable = ([]() {
   // The event only gets initialized in the entry point of the service. Child
   // processes and those run from the commandline will not have a stop event.
-  auto stopEvent = osquery::getStopEvent();
+  auto stopEvent = Initializer::isShell() ? nullptr : osquery::getStopEvent();
```

Comment:
Alternatively,
```cpp
if (Initializer::isShell()) {
  return;
}

Reply:
Lol, I had this, and then changed it because I figured you'd want a ternary :3

Reply:
Swapping out for that, as I kind of prefer the clarity of differentiation of running as the shell :D